src/main/java/net/preibisch/rsfish/spark/Block.java

```diff
@@ -52,12 +59,17 @@ public static ArrayList< Block > splitIntoBlocks( final Interval interval, final
 			long bs = blockSize[ d ];
 			long pos = interval.min( d );
 
-			while ( pos < interval.max( d ) - 1 )
+			while ( pos < interval.max( d ) + 1 )
```

Comment:
@StephanPreibisch this change I needed to be able to do channel and time index blocks